db.go

```diff
@@ -50,16 +52,54 @@ var migrations = &migrate.MemoryMigrationSource{
 				`,
 			},
 		},
+		{
+			Id: "3",
+			Up: []string{
+				`CREATE TABLE IF NOT EXISTS forwarding_history (
+                                        id INTEGER PRIMARY KEY,
```

Comment:
> Also I think that INTEGER PRIMARY KEY without AUTOINCREMENT may reuse deleted identifiers. 

Yeah [indeed](https://stackoverflow.com/questions/7905859/is-there-auto-increment-in-sqlite), I thought it was good _enough_  because it's not surfaced anywhere to end users. But it isn't really necessary (old habit from sql days of the past).

> possibly a CHECK constraint for the amount?
> And how about an index on add_time, to support the query function?

sgtm!